inspect.js

```diff
@@ -0,0 +1,16 @@
+const parse = require('.')
+const inspect = require('unist-util-inspect')
+const { hideBin } = require('yargs/helpers')
+const yargs = require('yargs/yargs')(hideBin(process.argv))
+
+const argv = yargs
+  .command('$0', 'Output the parsed syntax tree\n\nUsage: npm run inspect <message>')
+  .argv
+
+const message = argv._[0]
```

Comment:
yargs gives you some of this command handling behavior for free, I'll just send a follow up PR, once we merge this.